specs/carves.table

```diff
@@ -1,5 +1,5 @@
 table_name("carves")
-description("Forensic Memory Carves")
+description("Forensic File Carves.")
```

Comment:
I'd actually rather this be `Memory`, or maybe drop it and have just `Forensic carves.`, as the anticipation is that we're moving to process memory grabs next, and I'm anticipating having this all be in the same table.